src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.ts

```diff
@@ -217,4 +220,13 @@ export class AssessmentDashboardComponent implements OnInit, AfterViewInit {
     sortRows() {
         this.sortService.sortByProperty(this.exercises, this.exercisesSortingPredicate, this.exercisesReverseOrder);
     }
+
+    toggleSecondCorrection(exerciseId: number) {
+        const currentExercise = this.exercises.find((exercise) => exercise.id === exerciseId)!;
+        const index = this.exercises.indexOf(currentExercise);
+        this.exerciseService.toggleSecondCorrection(exerciseId).subscribe((res: HttpResponse<Boolean>) => {
```

Comment:
You are not doing any error handling here. You should catch the error here and display it in a alert component for example

Reply:
Done 👍 Thanks for catching that!